decidim-assemblies/config/locales/en.yml

```diff
@@ -48,6 +48,15 @@ en:
         title: Title
         twitter: Twitter
         youtube: YouTube
+      assembly_member:
+        birthday: Birthday
+        ceased_date: Ceased date
+        designation_date: Designation date
+        designation_mode: Designation mode
+        full_name: Full name
+        gender: Gender
+        origin: Origin
```

Comment:
@decidim/product what do we actually need to save here? Where the member is native from? See https://www.wordnik.com/words/origin

Reply:
@oriolgual sorry I forgot to answer this one. Yes, this refers to place of birth (again, data required from Barcelona Participation Regulations)

Reply:
Maybe `birthplace` would be a better naming?

Reply:
I'll be working on this today, I'll change this field to `birthplace`.